servers/v3-proxy-api/src/test/fixtures/fixtures.ts

```diff
@@ -103,7 +103,7 @@ type MockListItemObject = Omit<ListItemObject, 'item_id' | 'resolved_id'> & {
 /**
  * return REST v3 GET response for given Ids
  * //todo: need to include tags, images, videoes etc
```

Comment:
I actually did this but missed this comment

both this todo and the todo underneath are now done thanks to our combined efforts 💪

Reply:
YAY!